app/src/main/java/com/github/houseorganizer/houseorganizer/ShopListAdapter.java

```diff
@@ -0,0 +1,58 @@
+package com.github.houseorganizer.houseorganizer;
+
+import android.content.ClipData;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class ShopListAdapter extends RecyclerView.Adapter<ShopListAdapter.ItemsHolder> {
+
+    public class ItemsHolder extends RecyclerView.ViewHolder{
+        public CheckBox checkBox;
```

Comment:
Great idea to use a `CheckBox` button! I will probably switch the simple task buttons to this as well.